### DISCUSSION TITLE
TypeError: client.get(...).loginAs is not a function

### DISCUSSION THREAD

**Discussion by AbdellM** [adonisjs/core#4519] [Help] [answered]
I am trying to create a test senarior for login and I got an error with the loginAs funstion 
![Screenshot 2024-04-16 at 00 13 56](https://github.com/adonisjs/core/assets/71790032/d3f93522-a2a2-445e-ab18-23fcaea2f573)

From the Adonis doc https://docs.adonisjs.com/guides/http-tests#authenticating-users
![Screenshot 2024-04-16 at 00 24 40](https://github.com/adonisjs/core/assets/71790032/ec54da50-080c-49a3-8041-0185d6292029)

**Comment by RomainLanz** [comment, marked as answer]
Hey @AbdellM! 👋🏻 

Have you registered the `authApiClient()` Japa plugin in your `tests/bootstrap.ts` file?
